src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml

```diff
@@ -11,662 +11,340 @@
-          [1685009266.594973][39716:39718] CHIP:DMG: },
-          [1685009266.594976][39716:39718] CHIP:DMG:
-          [1685009266.594978][39716:39718] CHIP:DMG: },
-          [1685009266.594980][39716:39718] CHIP:DMG:
-          [1685009266.594982][39716:39718] CHIP:DMG: },
-          [1685009266.594985][39716:39718] CHIP:DMG:
-          [1685009266.594987][39716:39718] CHIP:DMG: ],
-          [1685009266.594990][39716:39718] CHIP:DMG:
-          [1685009266.594992][39716:39718] CHIP:DMG: InteractionModelRevision = 1
-          [1685009266.594994][39716:39718] CHIP:DMG: }
-      disabled: true
+      cluster: "Groups"
+      endpoint: Groups.Endpoint
+      command: "AddGroup"
+      arguments:
+          values:
+              - name: GroupID
+                value: 0x0104
+              - name: GroupName
+                value: ""
+      response:
+          error: UNSUPPORTED_ACCESS
 
     - label:
-          "TH sends a AddGroup Command to the Groups cluster with the GroupID
-          field set to 0x0104 and the GroupName set to an empty string. The
-          command is sent as a group command using GroupID 0x0103"
-      verification: |
-          ./chip-tool groups add-group 260 '' 0xffffffffffff0103 0
-      disabled: true
+          "Step 6: TH sends a AddGroup Command to the Groups cluster with the
+          GroupID field set to 0x0104 and the GroupName set to an empty string.
+          The command is sent as a group command using GroupID 0x0103"
+      cluster: "Groups"
+      endpoint: Groups.Endpoint
+      groupId: 0x0103
+      command: "AddGroup"
+      arguments:
+          values:
+              - name: GroupID
+                value: 0x0104
+              - name: GroupName
+                value: ""
+
+    # multicast if the unicast packet is sent immediately after the multicast one.
+    - label: "Wait for AddGroup"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 1000
 
     - label:
-          "TH sends a AddGroup Command to the Groups cluster with the GroupID
-          field set to 0x0105 and the GroupName set to an empty string. The
-          command is sent as a group command using GroupID 0x0104"
-      verification: |
-          ./chip-tool groups add-group 261 '' 0xffffffffffff0104 1
-      disabled: true
+          "Step 7: TH sends a AddGroup Command to the Groups cluster with the
+          GroupID field set to 0x0105 and the GroupName set to an empty string.
+          The command is sent as a group command using GroupID 0x0104"
+      cluster: "Groups"
+      endpoint: Groups.Endpoint
+      groupId: 0x0104
+      command: "AddGroup"
+      arguments:
+          values:
+              - name: GroupID
+                value: 0x0105
+              - name: GroupName
+                value: ""
+      response:
+          error: FAILURE
```

Comment:
The test plan does not say that this should result in FAILURE.  Why does this not match the test plan?  This is a group command; it won't know if it failed!

Reply:
@bzbarsky-apple Since step 10 states that step 7 was not successful, we consider it a failure.
![image](https://github.com/project-chip/connectedhomeip/assets/102359958/f8938d16-9c75-441d-bd86-61af3113c47f)

Reply:
It's not succcessful on the server @manjunath-grl , but it does not send a failure _response_, because it does not send any response at all, because it's a group command.

Reply:
Put another way, adding a "response" section to a group command is nonsense, and ideally would just cause the whole YAML parse to fail.  The fact that it does not is a bug; filed https://github.com/project-chip/connectedhomeip/issues/27967

Reply:
@bzbarsky-apple Thanks for the info. Updated script added prompt for steps 7 & 10 with issue.